common/src/main/scala/com/stratio/crossdata/common/Messages.scala

```diff
@@ -18,50 +18,53 @@ package com.stratio.crossdata.common
 
 import java.util.UUID
 
+import com.stratio.crossdata.common.result.SQLResult
 import com.stratio.crossdata.common.security.Session
-import org.apache.spark.sql.Row
 
 import scala.concurrent.duration.FiniteDuration
 
-trait Command
-
-case class SQLCommand(
-                       query: String,
-                       queryId: UUID = UUID.randomUUID(),
-                       retrieveColumnNames: Boolean = false,
-                       timeout: Option[FiniteDuration] = None
-                     ) extends Command {
+// Driver -> Server messages
+private[crossdata] trait Command {
+  def requestId: UUID
+}
 
-  def this(
-            query: String,
-            queryId: UUID,
-            retrieveColumnNames: Boolean,
-            timeout: FiniteDuration
-          ) = this(query, queryId, retrieveColumnNames, Option(timeout))
+private[crossdata] case class SQLCommand private(sql: String,
+                                                 requestId: UUID = UUID.randomUUID(),
+                                                 queryId: UUID = UUID.randomUUID(),
+                                                 retrieveColumnNames: Boolean = false,
```

Comment:
Lets use this PR to change this identifier to something more descriptive once for all. e.g: `flattenResults`